app/design/frontend/Gorban/Sergey/web/css/source/_buttons.less

```diff
@@ -0,0 +1,7 @@
+& when (@media-common = true) {
```

Comment:
There is such file already:
https://github.com/magento/magento2/blob/2.2.6/app/design/frontend/Magento/luma/web/css/source/_buttons.less